test/none.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-env mocha */
+
+var assert = require('assert')
+
+var util = require('./_util')
+var multer = require('../')
+var FormData = require('form-data')
+
+describe('None', function () {
+  var parser
+
+  before(function () {
+    parser = multer().none()
+  })
+
+  it('should not allow file uploads', function (done) {
+    var form = new FormData()
+
+    form.append('key1', 'val1')
+    form.append('key2', 'val2')
+    form.append('file', util.file('small0.dat'))
+
+    util.submitForm(parser, form, function (err, req) {
+      // "Unexpected field" error is expected
```

Comment:
Can you remove this comment and add `assert.ok(err)` as well so that it gives a better assertion error if no error was given. (as it is now it will give "Cannot read property code of undefined")